src/drivers/imu/mpu9250/mag.cpp

```diff
@@ -442,8 +456,8 @@ MPU9250_mag::passthrough_read(uint8_t reg, uint8_t *buf, uint8_t size)
 {
 	set_passthrough(reg, size);
 	usleep(25 + 25 * size); // wait for the value to be read from slave
```

Comment:
If the use of usleep() and similar is an issue, this may also be something to be reviewed (in passthrough_read() as well)